runtime/themes/yellowed.toml

```diff
@@ -0,0 +1,135 @@
+"punctuation"                     = { fg = "text" }
+"punctuation.special"             = { fg = "constants", modifiers = ["bold"] }
+"keyword"                         = { fg = "keywords", modifiers = ["bold"] }
+"keyword.control"                 = { fg = "keywords" }
+"keyword.control.return"          = { fg = "keywords", modifiers = ["bold"] }
+"keyword.operator"                = { fg = "keywords" }
+"keyword.directive"               = { fg = "keywords" }
+"operator"                        = { fg = "keywords" }
+"function"                        = { fg = "text", modifiers = ["bold"] }
+"tag"                             = { fg = "keywords", modifiers = ["bold"] }
+"namespace"                       = { fg = "text" }
+"special"                         = { fg = "yellow" }
+
+"markup.heading.marker"           = { fg = "keywords", modifiers = ["bold"] }
+"markup.heading.1"                = { fg = "text", modifiers = ["bold"] }
+"markup.heading.2"                = { fg = "text", modifiers = ["bold"] }
+"markup.heading.3"                = { fg = "text", modifiers = ["bold"] }
+"markup.heading.4"                = { fg = "text", modifiers = ["bold"] }
+"markup.heading.5"                = { fg = "text", modifiers = ["bold"] }
+"markup.heading.6"                = { fg = "text", modifiers = ["bold"] }
+"markup.list"                     = { fg = "keywords", modifiers = ["bold"] }
+"markup.bold"                     = { modifiers = ["bold"] }
+"markup.italic"                   = { modifiers = ["italic"] }
+"markup.link.url"                 = { fg = "text", modifiers = ["underlined"] }
+"markup.link.text"                = { fg = "string" }
+"markup.quote"                    = { fg = "text" }
+"markup.raw"                      = { fg = "string" }
+
+"diff.plus"                       = { fg = "green" }
+"diff.minus"                      = { fg = "red" }
+"diff.delta"                      = { fg = "blue" }
+"diff.delta.moved"                = { fg = "gray" }
+
+"ui.background"                   = { bg = "dark_gray", fg = "text" }
+"ui.background.separator"         = { fg = "text" }
+"ui.cursor"                       = { bg = "yellow", fg = "black" }
+"ui.cursor.normal"                = { bg = "yellow", fg = "black" }
+"ui.cursor.insert"                = { bg = "yellow", fg = "black" }
+"ui.cursor.select"                = { bg = "yellow", fg = "black" }
+"ui.cursor.match"                 = { bg = "light_gray" }
+"ui.cursor.primary"               = { bg = "yellow", fg = "black" }
+"ui.cursor.primary.normal"        = { bg = "yellow", fg = "black" }
+"ui.cursor.primary.insert"        = { bg = "yellow", fg = "black" }
+"ui.cursor.primary.select"        = { bg = "yellow", fg = "black" }
+"ui.debug.breakpoint"             = { fg = "red" }
+"ui.debug.active"                 = { fg = "green" }
+"ui.gutter"                       = { fg = "light_gray" }
+"ui.gutter.selected"              = { fg = "light_gray" }
+"ui.highlight.frameline"          = { fg = "cursor_line" }
```

Comment:
It looks like this color doesn't exist in the palette

Reply:
Old color, fixed